packages/react-instantsearch-hooks/src/usePoweredBy.ts

```diff
@@ -0,0 +1,12 @@
+import { safelyRunOnBrowser } from 'instantsearch.js/es/lib/utils';
```

Comment:
not 100% sure we can rely on internals like this, changing the name or export isn't a breaking change

Reply:
Since we're in control of the IS.js version used, I think it's fine?

Reply:
AFAIK we don't have a common source of helpers, so we can probably copy `safelyRunOnBrowser` in the codebase for now and refactor later on. What do you think?